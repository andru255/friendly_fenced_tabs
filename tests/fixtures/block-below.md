# my title

A single text

## now a subtitle

An other text

```
my block below
```