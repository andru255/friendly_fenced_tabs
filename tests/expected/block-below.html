<h1>my title</h1>
<p>A single text</p>
<h2>now a subtitle</h2>
<p>An other text</p>
<pre>
    my block below
</pre>